themes/src/main/resources/theme/base/account/messages/messages_en.properties

```diff
@@ -6,6 +6,8 @@ doAdd=Add
 doSignOut=Sign Out
 doLogIn=Log In
 doLink=Link
+doDelete=Delete
```

Comment:
@andymunro  can you review these changes? Also see email "KEYCLOAK-953: allow user to delete account" in keycloak-dev ML for screenshots.